### DISCUSSION TITLE
VcTerrainProviderTianditu组件，1.92以前的版本出现了跨域错误

### DISCUSSION THREAD

**Discussion by Asakun** [zouyaoji/vue-cesium#567] [Q&A]
1.92+的版本确实和文档说的一样，会直接报错用不了。
但我用的https://unpkg.com/cesium@1.91/Build/Cesium/Cesium.js 现在出现了跨域问题
<img width="1219" alt="Screenshot 2024-06-05 at 10 06 20" src="https://github.com/zouyaoji/vue-cesium/assets/4018751/e6f49342-fe4c-4b56-a28f-72c8b9528288">
我的浏览器是用的125.0.2535.51 的edge，在safari中也有同样的问题
在一个月前，我编写自己的demo的时候VcTerrainProviderTianditu组件还是可以正常使用的，可以完成地形加载。
不知道是天地图的接口更新了，还是浏览器更新导致的问题。
另外文档中案例同样会跨域报错，不能加载地形

**Comment by zouyaoji** [comment]
这儿都出不来了，暂时不知道咋搞
http://lbs.tianditu.gov.cn/docs/#/sanwei/
